.github/workflows/pull-request.yml

```diff
@@ -83,6 +84,27 @@ jobs:
       MONGO_DB_URL: mongodb://localhost:27017/talawa-test-db
       REDIS_HOST: localhost
       REDIS_PORT: 6379
+      RECAPTCHA_SITE_KEY: "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+      RECAPTCHA_SECRET_KEY: "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+      MAIL_USERNAME: "abc@gmail.com"
+      MAIL_PASSWORD: "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+      IS_SMTP: ""
+      SMTP_HOST: ""
+      SMTP_PASSWORD: ""
+      SMTP_USERNAME: ""
+      messagingSenderId: "XXXXXXXXXXXX"
+      projectId: "appname-XXXXXX"
+      storageBucket: "appname-XXXXXX.appspot.com"
+      iOSapiKey: "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+      iOSappId: "1:XXXXXXXXXXXX:ios:XXXXXXXXXXXXXXXXXXXXXX"
```

Comment:
These are no longer used in the `.env` file
![image](https://github.com/PalisadoesFoundation/talawa-api/assets/16875803/3dbc5969-18bd-4ff6-a405-400a0f4da3df)

Reply:
There are no tokens defined:
- ACCESS_TOKEN_SECRET
- REFRESH_TOKEN_SECRET

Reply:
Have you tried starting the server with these `.env` values on your local system?